backend/models/index.js

```diff
@@ -1,25 +1,34 @@
-const dbConfig = require("../config/db.config.js");
-const Sequelize = require("sequelize");
-
-// CREATE SEQUELIZE INSTANCE
-const db = {};
-
-// DEFINE SEQUALIZE INSTANCE
-// SEQUELIZE- Option 3: Passing parameters separately (other dialects)
-const sequelize = new Sequelize(dbConfig.DB, dbConfig.USER, dbConfig.PASSWORD, {
-	host: dbConfig.HOST,
-	dialect: dbConfig.dialect,
-
-	pool: {
-		max: dbConfig.pool.max,
-		min: dbConfig.pool.min,
-		acquire: dbConfig.pool.acquire,
-		idle: dbConfig.pool.idle,
-	},
-});
-
-// RETURN SEQUELIZE INSTANCE
-db.sequelize = sequelize;
-db.Sequelize = Sequelize;
-
+const {Sequelize, DataTypes} = require('sequelize');
+
+//connecting to database
+const sequelize = new Sequelize('testdb', 'admin', 'admin123',{
+                                host: 'localhost',
+                                dialect: 'mysql',
+                                //define pool on release lvl
+});
+
+try{
+    sequelize.authenticate();
+    console.log('connected...');
+}catch(err){
+    console.error('unable to connect', err);
+}
+
+const db = {};
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
+
+//initialize all entity in DB

```

Comment:
do Sequelize need manual initialization?

Reply:
this is to pass that Sequelize library to object db. so everytime we want to use object db in another module, no need to require Sequelize library again, only require the object